datafusion/proto/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -31,8 +31,10 @@ use datafusion::datasource::TableProvider;
 use datafusion::execution::context::SessionState;
 use datafusion::execution::runtime_env::{RuntimeConfig, RuntimeEnv};
 use datafusion::execution::FunctionRegistry;
-use datafusion::functions_aggregate::covariance::{covar_pop, covar_samp};
-use datafusion::functions_aggregate::expr_fn::first_value;
+use datafusion::functions_aggregate::expr_fn::{
```

Comment:
Could we add some documentation with examples somewhere? That would both serve to help document and test this feature.

But thank you @jayzhan211  -- this is looking very cool